BingMaps/v8-web-control/modules/venue-map-module/venuemapfootprintslayeroptions-object.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Venue Maps Options Object | Microsoft Docs"
+ms.custom: ""
+ms.date: "06/12/20"
+ms.reviewer: ""
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+ms.assetid: ""
+caps.latest.revision: 0
+author: "simshap"
+ms.author: "simshap"
+manager: "cordellj"
+ms.service: "bing-maps"
+---
+
+# VenueMapFootprintsLayerOptions
+Options used to customize how a [venueTile](../../venues/venueTile.md) file is read and loaded via the [VenueMapFactory](venuemapfactory-class.md).
+
+When hosting venue map footprints layer metadata at a custom endpoint, one needs to ensure that the data hosted at `tileUrl` can be loaded using the method defined in `tileLoader`. Specifically, one should pay attention to the case where their data is hosted on a different domain where CORS is not enabled. In this case, the hosting site should create another webpage triggered by an additional parameter (referred to below as the JSONP URL parameter) which allows for the file to be downloaded across domains.
+
+
+## Properties
```

Comment:
All properties of VenueMapFootprintsLayerOptions are optional, though one of tileUrl | tileLoader must be present.